src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -256,10 +313,16 @@ class CoreWorkerDirectTaskSubmitter {
 
     // Check whether the pipelines to the active workers associated with a
     // SchedulingKeyEntry are all full.
-    bool AllPipelinesToWorkersFull(uint32_t max_tasks_in_flight_per_worker) const {
+    inline bool AllPipelinesToWorkersFull(uint32_t max_tasks_in_flight_per_worker) const {
       return total_tasks_in_flight ==
```

Comment:
```suggestion
      return total_tasks_in_flight >=
```

(just a defensive change).